### DISCUSSION TITLE
elementary OS 8 fails to boot with AMD GPUs only showing blank screen

### DISCUSSION THREAD

**Discussion by vjr** [elementary/triage#633] [Q&A] [answered]
After installing elementary OS 8 you may have failure to fully boot and only see blank screen on hardware with AMD GPUs.

**Comment by vjr** [comment, marked as answer]
You have two options... either permanently remove the splash option by removing it from the `/etc/default/grub` file then running `sudo update-grub` or, if you really want to preserve working splash then do the following:

Edit `/etc/initramfs-tools/modules` and add a line `amdgpu` to it and save, then run `sudo update-initramfs -uk all` to redo the kernel so that upon next boot it "should" properly display the boot splash logo/animation... 🙂

This will be preserved going forward even when there are new kernel updates - thanks to the addition of the `amdgpu` entry into the `/etc/initramfs-tools/modules` file.

HTH!

**Comment by marekpszczolka** [comment]
Good evening,
After installing Elementary OS 8, "Display Mirroring" does not work for me when I use an external monitor with my laptop. The system sees the monitor, the wallpaper appears on the monitor, but when I enable mirroring in the display settings, the setting is turned off. In other systems - Debian, Ubuntu, Linux Mint, Zorin OS, display mirroring always works for me. Can this error be fixed?
My laptop configuration is:
marp@elementaryOS
OS: elementary OS trixie/sid (circe) x86_64 
Host: 82XM IdeaPad Slim 3 15ABR8 
Kernel: 6.8.0-49-generic 
Packages: 1832 (dpkg), 10 (steam), 88 (flatpak-system) 
Shell: bash 5.2.21 
Resolution: 1920x1080, 1920x1080 
DE: Pantheon (wayland) 
WM: gala 
WM Theme: Adwaita 
Terminal: io.elementary.terminal 
CPU: AMD Ryzen 7 7730U (16) @ 4.5GHz 
Memory: 3.04 GiB / 13.48 GiB (22%) 
Bluetooth: Foxconn / Hon Hai MediaTek Bluetooth Adapter 
BIOS: LENOVO 1.28 (11/07/2023)
